client/packages/common/src/intl/locales/en/common.json

```diff
@@ -443,6 +443,9 @@
   "heading.details": "Details",
   "heading.devices": "Devices",
   "heading.dispensary-details": "Dispensary Details",
+  "heading.pricing": "Pricing",
```

Comment:
Pricing is in the new designs, and section was internally labeled as pricing, so I added the translation here.